### ISSUE TITLE
Cleanup feature not working

### ISSUE THREAD

**Issue by StanESmith** [dvehrs/podget#37]
Having trouble running the cleanup option of podget. When I run `podget -C`the return is "No Session File Found" and nothing gets deleted. The permissions are setup correctly and I specified a sessions directory  but still the cleanup does not work. Any direction on where I should look?

**Comment by dvehrs** [comment]
First allow me to apologize for taking so long to respond.  A combination of being busy and this issue requiring a slightly more complicated fix than anticipated delayed my response.

I've uploaded a potential fix to the DEV branch.  Feel free to test it.

The problem was caused by the simple fact that the Cleanup Loop had not been updated in ages and did not anticipate how we create and update playlists today.  When it was implemented, playlists were meant to be created each time podget ran and only include the files added on that day.  Playlists today are often configured to be separate for each feed and are updated each time podget runs to show all files available for that feed.  So the old test looked for playlists with a modification date older than CLEANUP_DAYS.  Well this would never be found because all playlists were recreated every time podget ran.  To fix this I changed the loop to look for individual files on each playlist that are older than CLEANUP_DAYS and when found to remove them.  Then at the end, playlists are then recreated to show the files that remain.

Also, I changed how the date is calculated in a small way.  In the past, it was calculated so 1 day was equal to 24 hrs from the time the script was run.  OK if your scheduling it, but not OK for general purposes.  I changed the date calculation to start from midnight of the day the script is run (date --date 0:00).  This means for the purposes of cleanup, if we set CLEANUP_DAYS to 1, then all files that are older than midnight yesterday will be removed.  Or if we set CLEANUP_DAYS to 0 then all files that are older than midnight last night will be removed.  

As, I mention this fix is now part of the DEV branch and ready for testing.  Have fun and let me know if it works for you or if there is anything you would change.

Dave
